keywords/TestServeriOS.py

```diff
@@ -28,16 +27,29 @@ def __init__(self, version_build, host, port, community_enabled=None, debug_mode
         self.debug_mode = False
         self.app_path = ""
         self.app_name = ""
+        self.platform = platform
+        self.bundle_id = ""
+
         if debug_mode:
             self.debug_mode = True
-        if community_enabled:
-            self.app_dir = "CBLTestServer-iOS-community-{}".format(version_build)
-            self.package_name = "CBLTestServer-iOS-community-{}.zip".format(version_build)
-            self.app = "CBLTestServer-iOS"
+
+        if self.platform == "ios":
+            if community_enabled:
+                self.app_dir = "CBLTestServer-iOS-community-{}".format(version_build)
+                self.package_name = "CBLTestServer-iOS-community-{}.zip".format(version_build)
+                self.app = "CBLTestServer-iOS"
+            else:
+                self.app_dir = "CBLTestServer-iOS-enterprise-{}".format(version_build)
+                self.package_name = "CBLTestServer-iOS-enterprise-{}.zip".format(version_build)
+                self.app = "CBLTestServer-iOS-EE"
+
+            self.bundle_id = "com.couchbase.CBLTestServer-iOS"
         else:
-            self.app_dir = "CBLTestServer-iOS-enterprise-{}".format(version_build)
-            self.package_name = "CBLTestServer-iOS-enterprise-{}.zip".format(version_build)
-            self.app = "CBLTestServer-iOS-EE"
+            # Xamarin-ios
+            self.app_dir = "TestServer.iOS"
+            self.package_name = "TestServer.iOS.zip"
+            self.app = "TestServer.iOS"
+            self.bundle_id = "com.couchbase.TestServer-iOS"
```

Comment:
self.bundle_id is same for both if and else condition, you may want to put it outside of if else?

Reply:
They are not the same, CBLTestServer-iOS vs TestServer-iOS